db/seeds.d/106-mail_notifications.rb

```diff
@@ -31,6 +31,13 @@
      :mailer => 'Katello::ErrataMailer',
      :method => 'promote_errata',
      :subscription_type => 'alert'
+    },
+
+    {:name => :subscription_expiry,
```

Comment:
Prefer the name to be `:subscriptions_expiring_soon` (it seems this value is capitalized and used as the name in the UI.)